File: psipuc/src/java/puc/si/psi/ti/model/Doador.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package puc.si.psi.ti.model;

import java.util.List;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

/**
 *
 * @author Wilton Garcia
 */
@Entity
public class Doador {
    @Id
    @GeneratedValue
    private int idDoador;
    private String emailDoador;
    private String cpfDodaor;
    private String senhaDoador;
    private List<Doacao> historicoDoador;

    public int getIdDoador() {
        return idDoador;
    }

    public void setIdDoador(int idDoador) {
        this.idDoador = idDoador;
    }

    public String getEmailDoador() {
        return emailDoador;
    }

    public void setEmailDoador(String emailDoador) {
        this.emailDoador = emailDoador;
    }

    public String getCpfDodaor() {
        return cpfDodaor;
    }

    public void setCpfDodaor(String cpfDodaor) {
        this.cpfDodaor = cpfDodaor;
    }

    public String getSenhaDoador() {
        return senhaDoador;
    }

    public void setSenhaDoador(String senhaDoador) {
        this.senhaDoador = senhaDoador;
    }

    public List<Doacao> getHistoricoDoador() {
        return historicoDoador;
    }

    public void setHistoricoDoador(List<Doacao> historicoDoador) {
        this.historicoDoador = historicoDoador;
    }
    
    
    
}

File: PucPsi6Php/model/Doador.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Doador
 *
 * @author Wilton Garcia
 */
class Doador {
    private $idDoador;
    private $emailDoador;
    private $senhaDoador;
    
    private $nomeDoador;
    private $icpfDoador;
    private $historicoDoador = [];
    
    function getIdDoador() {
        return $this->idDoador;
    }

    function getEmailDoador() {
        return $this->emailDoador;
    }

    function getSenhaDoador() {
        return $this->senhaDoador;
    }

    function getNomeDoador() {
        return $this->nomeDoador;
    }

    function getIcpfDoador() {
        return $this->icpfDoador;
    }

    function getHistoricoDoador() {
        return $this->historicoDoador;
    }

    function setIdDoador($idDoador) {
        $this->idDoador = $idDoador;
    }

    function setEmailDoador($emailDoador) {
        $this->emailDoador = $emailDoador;
    }

    function setSenhaDoador($senhaDoador) {
        $this->senhaDoador = $senhaDoador;
    }

    function setNomeDoador($nomeDoador) {
        $this->nomeDoador = $nomeDoador;
    }

    function setIcpfDoador($icpfDoador) {
        $this->icpfDoador = $icpfDoador;
    }

    function setHistoricoDoador($historicoDoador) {
        $this->historicoDoador = $historicoDoador;
    }



}
